third_party/asylo/status_macros.h

```diff
@@ -0,0 +1,85 @@
+/*
+ *
+ * Copyright 2018 Asylo authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef THIRD_PARTY_ASYLO_STATUS_MACROS_H_
+#define THIRD_PARTY_ASYLO_STATUS_MACROS_H_
+
+#include "absl/base/optimization.h"
+
+/// Evaluates an expression that produces an `Status`-like object with
+/// a `.ok()` method. If this method returns false, the object is
+/// returned from the current function.
+///
+/// Example:
+/// ```
+///   ::absl::Status MultiStepFunction() {
+///     OAK_RETURN_IF_ERROR(Function(args...));
+///     OAK_RETURN_IF_ERROR(foo.Method(args...));
+///     return ::absl::OkStatus();
+///   }
+/// ```
+#define OAK_RETURN_IF_ERROR(expr)                          \
```

Comment:
Will this rename as a separate commit in the same PR be an issue when the PR is squashed and merged? Or is the plan to merge without squashing?

Reply:
I'll rebase-and-merge.